google/cloud/storage/internal/object_streambuf.cc

```diff
@@ -332,7 +336,9 @@ StatusOr<HttpResponse> ObjectWriteStreambuf::Flush() {
   StatusOr<ResumableUploadResponse> result =
       upload_session_->UploadChunk(current_ios_buffer_);
   if (!result) {
-    // This was an unrecoverable error, time to signal an error.
+    // This was an unrecoverable error, time to store status and signal an
+    // error.
+    last_status_ = result.status();
```

Comment:
See below, I think this should be `last_response_ = result.status()`.

Reply:
done.